joern-cli/frontends/javasrc2cpg/src/test/scala/io/joern/javasrc2cpg/querying/CallTests.scala

```diff
@@ -334,6 +334,8 @@ class CallTests extends JavaSrcCodeToCpgFixture {
 
     argument.name shouldBe Operators.fieldAccess
     argument.typeFullName shouldBe "test.MyObject"
+    argument.order shouldBe 1
```

Comment:
For easier readability please also check for `argument.code shouldBe "obj"`.

Reply:
Will do!